factory/installScripts/edge/default.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+# Microsoft offers a debian package, here we're adding the package list and then installing the specific version we want to install.
+apt-get update \
+  && apt-get install --no-install-recommends -y \
+    ca-certificates \
+  && curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg \
+  && install -o root -g root -m 644 microsoft.gpg /etc/apt/trusted.gpg.d \
+  && sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/edge stable main" > /etc/apt/sources.list.d/microsoft-edge-stable.list' \
```

Comment:
Two notes here, we're now installing a specific version of edge and we are using the stable version instead of the dev version